src/shifter.c

```diff
@@ -358,6 +365,11 @@ int parse_options(int argc, char **argv, struct options *config, UdiRootConfig *
                 }
                 break;
             case 'w':
+                if (config->workdir) {
+                    free(config->workdir);
+                    config->workdir = NULL;
+                }
+                config->useWorkDir = 1;
```

Comment:
I'm wondering if we should also enable this if --entry is specified since the entrypoint could be a relative path.

Reply:
Disregard.  I think this is fine as is.